wallet/src/main/java/com/android/identity_credential/wallet/presentation/OpenID4VPPresentationActivity.kt

```diff
@@ -761,6 +811,7 @@ private suspend fun getAuthorizationRequest(
             clientId = requestUri.getQueryParameter("client_id")!!,
             nonce = requestUri.getQueryParameter("nonce")!!,
             responseUri = requestUri.getQueryParameter("response_uri")!!,
+            responseMode = requestUri.getQueryParameter("response_mode")!!,
```

Comment:
So the response mode can't be null here? I have no full picture, but in 2 places above I saw that the null case for it treated (with when()) and no null case treated (if else). Could be unrelated to this check but might be worth investigating.